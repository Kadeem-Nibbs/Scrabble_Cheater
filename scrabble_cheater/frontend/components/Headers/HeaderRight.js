import './HeaderRight.less'

import React, { Component } from 'react'
import { Label, Icon } from 'semantic-ui-react'

class HeaderRight extends Component {
  render() {  
    return(
      <Label className="header-right"> 
        <Icon name='info' /> Enter <span className="big-underscore">_</span> for blank tiles. 
        <div>Blank tiles on the board will be above a <span className="big-underscore">_</span>.</div>
      </Label>
    )
  } 
}

export default HeaderRight

