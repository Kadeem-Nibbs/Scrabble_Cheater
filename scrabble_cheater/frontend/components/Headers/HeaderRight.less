.header-right {
  .big-underscore {
    font-size: 2rem;
    position: relative;
    top: -3px;
    margin-left: 3px;
    margin-right: 3px;
    line-height: 1px;
  }
}