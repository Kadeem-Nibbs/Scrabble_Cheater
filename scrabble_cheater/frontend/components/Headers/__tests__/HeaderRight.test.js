import React from 'react'
import HeaderRight from '../HeaderRight'

test('render HeaderRight', () => {
    const wrapper = shallow(<HeaderRight />)
    expect(wrapper).toMatchSnapshot()    
});
