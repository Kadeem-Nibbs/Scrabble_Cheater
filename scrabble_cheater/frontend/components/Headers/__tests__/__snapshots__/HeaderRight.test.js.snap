// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render HeaderRight 1`] = `
<Label
  className="header-right"
>
  <Icon
    as="i"
    name="info"
  />
   Enter 
  <span
    className="big-underscore"
  >
    _
  </span>
   for blank tiles.
  <div>
    Blank tiles on the board will be above a 
    <span
      className="big-underscore"
    >
      _
    </span>
    .
  </div>
</Label>
`;
